spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -481,16 +481,19 @@ def load_figure(self, fig, fmt):
         # immediately after the figure is loaded doesn't work.
         QTimer.singleShot(
             20,
-            lambda: self.verticalScrollBar().setValue(
-                self.current_thumbnail.vscrollbar_value
-            ),
+            self.update_scrollbars_values,
         )
-        QTimer.singleShot(
-            20,
-            lambda: self.horizontalScrollBar().setValue(
+
+    def update_scrollbars_values(self):
+        try:
```

Comment:
Please add a comment on top of this line to explain why the try/except is necessary.

Reply:
Checking this again I think the try/except is not necessary so I removed it :+1: